lib/boxes.js

```diff
@@ -23,7 +23,18 @@ async function setBoxByTitle(title, item) {
   return postBox;
 }
 
+async function setItemByTitle(title, item) {
+  const addItem = await collection("boxes").updateOne(
```

Comment:
The result of `updateOne` is not the box, it's [this](https://mongodb.github.io/node-mongodb-native/3.6/api/Collection.html#~updateWriteOpResult).

```suggestion
  const updateResult = await collection("boxes").updateOne(
```

This allows to check if something was updated. See next comment.